v1/contracts/Voting.sol

```diff
@@ -0,0 +1,42 @@
+pragma solidity ^0.5.0;
+
+
+/**
+ * @title Voting system for Oracle.
+ * @dev Handles receiving and resolving price requests via a commit-reveal voting scheme.
+ */
+contract Voting {
+
+    struct VoteInstance {
+        // Maps (voterAddress) to their committed hash.
+        // A bytes32 of `0` indicates no commit or a commit that was already revealed.
+        // TODO(ptare): Do we prefer to store an extra boolean instead?
```

Comment:
IMO, this is way better than storing an accompanying boolean.

Reply:
Sounds good, removed this TODO.